docs/streams/upgrade-guide.html

```diff
@@ -74,7 +74,6 @@ <h1>Upgrade Guide and API Changes</h1>
 
     <h3><a id="streams_api_changes_240" href="#streams_api_changes_240">Streams API changes in 2.4.0</a></h3>
 
-    <!-- Placeholder KIP-213 -->
```

Comment:
Why was the text not inserted here? I think that the new join is the coolest new feature and should be listed first :)

Reply:
I had it there, then it moved. I'll put it back there.